### PR TITLE
Start the titles-for-developers.md file

### PR DESCRIPTION
**Pull request by bemurphy**
base this off a structure and content style similar to Basecamp:

https://github.com/basecamp/handbook/blob/master/titles-for-programmers.md

### CODE REVIEW

File: titles-for-developers.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+* Capable of handling customer support, maintenance, and application operational issues with little
+  assistance.
+* Able to interface with other departments and leaders in company, as well as other vendors, at times when required.
+* Usually at least 3-7 years of experience being a professional developer in SaaS web applications.
+
+## Lead Developer
+
+* Expected to run and direct small teams for project cycles.
+* Deep understanding of Ruby and Rails environments
+* Expected to provide feedback an assist with the development of junior developers and developers.
```
Comment (nobodyiscertain) [inline titles-for-developers.md]: `Expected to provide feedback an assist` I think that `an` should be an `and`

File: titles-for-developers.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+* Fully capable to architect novel system on top of technology not provided by a framework (ex: building a distributed
```
Comment (nobodyiscertain) [inline titles-for-developers.md]: `Fully capable to architect novel system on` Think that should be `a novel system` or `novel systems`

File: titles-for-developers.md (hunk 1 above)
Comment (bemurphy) [reply titles-for-developers.md]: good catch, fixed!

File: titles-for-developers.md (hunk 2 above)
Comment (bemurphy) [reply titles-for-developers.md]: Yup, good catch.  I changed to "novel systems"

**Comment by brentd** [comment]
I like this! removing the `review` label.

**Comment by snelson** [comment]
This has been updated a bit and will be added to our notion docs.

**Pull request closed without merging**